src/scripts/browserDeviceProfile.js

```diff
@@ -194,7 +194,8 @@ function supportsHdr10(options) {
             || browser.web0s
             || browser.safari && ((browser.iOS && browser.iOSVersion >= 11) || browser.osx)
             // Chrome mobile and Firefox have no client side tone-mapping
-            // Edge Chromium on Nvidia is known to have color issues on 10-bit video
+            // Edge Chromium 121+ fixed the tone-mapping color issue on Nvidia
+            || browser.edgeChromium && (browser.versionMajor >= 121)
```

Comment:
I would split the commit. This change doesn't apply to Safari.

Reply:
Done.